test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -154,6 +154,19 @@ def test_vol(self, gain, gain_type):
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform])
 
+    @parameterized.expand([
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': False, 'norm_vars': False}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': True, 'norm_vars': False}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': False, 'norm_vars': True}, ),
+        ({'cmn_window': 600, 'min_cmn_window': 100, 'center': True, 'norm_vars': True}, ),
+        ({'cmn_window': 500, 'min_cmn_window': 50, 'center': False, 'norm_vars': False}, ),
```

Comment:
`cmn_window =600` and `min_cmn_window=100` look too big for the input with `8000 * 0.05 == 400` (then FFT applied) can you make them somewhat smaller than the number of frames in time axis of the input tensor?

Reply:
thanks @mthrok will try to implement your suggestions